### PR TITLE
Add note about the pang12/13 switch

### PR DESCRIPTION
**Pull request by ahoneybun**

### CODE REVIEW

**Review by jacobgkau — DISMISSED** [review]
I rewrote the article to better incorporate the new information and fix grammatical errors. It looks good to me now; support should review before merging.

**Review by jacobgkau — APPROVED** [review]

**Review by thomas-zimmerman — COMMENTED** [review]
Most of the changes look good. The Pangolin modes should be a note not another bullet point (and the final bullet point is indented):
image

**Review by fs1995 — APPROVED** [review]

**Pull request merged**
